media-plugins/vdr-ddci2/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>d.scheller.oss@gmail.com</email>
+		<name>Daniel Scheller</name>
+	</maintainer>
+	<upstream>
```

Comment:
Thanks for pointing this out, will do next time (as mentioned, first-time contribution :) ) - and thanks for merging!